src/main/java/com/backbase/eo/testing/events/emitter/RawEmittingController.java

```diff
@@ -57,6 +60,8 @@ public ResponseEntity<Void> emitEvent(@RequestBody RawEventPayload payload) {
         envelopedEvent.setEvent(payload.getBody());
 
         MessageBuilder eventMessageBuilder = MessageBuilder.withPayload(payload.getBody()).setHeader("bbEventType", payload.getEventType());
+        ofNullable(payload.getCustomHeadersValues()).orElse(emptyMap()).forEach(eventMessageBuilder::setHeader);
```

Comment:
it will add new headers
existing headers will be overwritten only if we set the same header name in the custom headers.
Let me explicitly cover this case in the tests